### ISSUE TITLE
DB InnerSource Story

### ISSUE THREAD

**Issue by rrrutledge** [InnerSourceCommons/InnerSourceMarketing#336]
See this [LinkedIn Post](https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fcomm%2Ffeed%2Fupdate%2Furn%253Ali%253Aactivity%253A7186966337678684162%3FcommentUrn%3Durn%253Ali%253Acomment%253A%2528activity%253A7186966337678684162%252C7187003289861828609%2529%26dashCommentUrn%3Durn%253Ali%253Afsd_comment%253A%25287187003289861828609%252Curn%253Ali%253Aactivity%253A7186966337678684162%2529%26lipi%3Durn%253Ali%253Apage%253Aemail_email_notification_single_mentioned_you_in_this_01%253BvruI7xvARMe%252BV77HwHn5Jg%253D%253D%26midToken%3DAQGGeYPL9IT-GQ%26midSig%3D3qiMaUu-TYNbc1%26trk%3Deml-email_notification_single_mentioned_you_in_this_01-hero_notification_cta-0-1ep~cta%26trkEmail%3Deml-email_notification_single_mentioned_you_in_this_01-hero_notification_cta-0-1ep~cta-null-hpjzw~lv6eoums~w8-null-null%26eid%3Dhpjzw-lv6eoums-w8&data=05%7C02%7C%7C6be35b0a9f1142fc3abd08dc604aad2d%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638491121059444231%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=yEFHK3qeTDLRaF40rP07I7KVNsKAS%2Bnu9CEVHJ6rblI%3D&reserved=0).

**Comment by Maryblessing** [comment]
@rrrutledge a bit of clarity. did we want to add their InnerSource story to the website? if yes, can you confirm I can now close this issue?

**Comment by rrrutledge** [comment]
Yes!  We want to add InnerSource stories to the website.  Please do!
